services/PostService.js

```diff
@@ -0,0 +1,21 @@
+const mongoose      = require('mongoose');
+const Post          = require('../models/Post');
+
+mongoose.Promise = require('bluebird');
+mongoose.connect('mongodb://test:test@ds257627.mlab.com:57627/first-blog-mern', { useMongoClient: true });
```

Comment:
I'd suggest you create a mongoose.js that initializes mongoose and also exports mongoose, and in all your models, instead of requiring 'mongoose' require './mongoose' (or something similar)

Reply:
fixed